Gemfile.lock

```diff
@@ -212,7 +212,9 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.5)
+    mimemagic (0.3.10)
```

Comment:
Apparently, this is not enough as the library now has a dependency over [`shared-mime-info`](https://pkgs.alpinelinux.org/package/edge/main/x86/shared-mime-info) package.